openstack.md

```diff
@@ -3,101 +3,48 @@ OpenStack Integration
 
 OPS works perfectly fine with OpenStack.
 
-For most operations you'll want to use the following config and you'll
-need a handful of env vars.
-
-```sh
-package main
-
-import (
-  "fmt"
-  "net/http"
-)
-
-func main() {
-  fmt.Println("hello world!")
-
-  http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
-    fmt.Fprintf(w, "Welcome to my website!")
-  })
-
-  fs := http.FileServer(http.Dir("static/"))
-  http.Handle("/static/", http.StripPrefix("/static/", fs))
-
-  http.ListenAndServe("0.0.0.0:8080", nil)
-}
+Some cloud providers using OpenStack that have been tested are [vexxhost](https://vexxhost.com/) and [ovh](https://www.ovh.com/).
+
+In the cloud provider, download a script with the env variables required.
+1. Create an account;
+2. Login;
+3. Click `API Access` in left menu;
+4. Click on button with the description `Download OpenStack RC File`;
+5. Select an option which contains the description `RC File`.
+![download rc file](openstack-rc-file.png)
+6. Run `RC file` script in the terminal where you will use ops.
+```
+$ ./6050015361872676-openrc.sh`
 ```
-
-These examples have been tested out on vexxhost.net:
 
 ## Image Operations
 ### Create Image
 
 ```sh
-#!/bin/sh
-
-export OS_USERNAME=""
-export OS_PASSWORD=""
-export OS_DOMAIN_NAME="Default"
-export OS_AUTH_URL="https://auth.vexxhost.net/"
-export OS_REGION_NAME="sjc1"
-export OS_PROJECT_NAME=""
-export OS_PROJECT_ID=""
-
-GOOS=linux go build -o gtest
+GOOS=linux go build -o <image_name>
 
-ops image create -c config.json -t openstack -a gtest
+ops image create -c config.json -t openstack -a <image_name>
 ```
 
 ### List Images
 
 ```sh
-#!/bin/sh
-
-export OS_USERNAME=""
-export OS_PASSWORD=""
-export OS_DOMAIN_NAME="Default"
-export OS_AUTH_URL="https://auth.vexxhost.net/"
-export OS_REGION_NAME="sjc1"
-export OS_PROJECT_NAME=""
-export OS_PROJECT_ID=""
-
 ops image list -t openstack
 ```
 
 ### Delete Image
 
 ```sh
 
-#!/bin/sh
-
-export OS_USERNAME=""
-export OS_PASSWORD=""
-export OS_DOMAIN_NAME="Default"
-export OS_AUTH_URL="https://auth.vexxhost.net/"
-export OS_REGION_NAME="sjc1"
-export OS_PROJECT_NAME=""
-export OS_PROJECT_ID=""
-
-ops image delete -t openstack gtest
+ops image delete -t openstack <image_name>
 ```
 
 ## Instance Operations
 ### Create Instance
 
 ```sh
 
-#!/bin/sh
```

Comment:
this is all good but should we leave the env var way as an alt method?

Reply:
I added the option of setting env variables manually on the top of the page. Let me know if it is more intuitive to have it replicated in each action code.

Reply:
nah that's fine